src/caret_analyze/value_objects/message_context.py

```diff
@@ -124,6 +124,14 @@ def subscription_topic_name(self) -> Optional[str]:
             return None
         return self._sub.topic_name
 
+    @property
+    def publisher_construction_order(self) -> Optional[int]:
+        return None
```

Comment:
Need to return construction_order of publisher.
```suggestion
        if self._pub is None:
            return None
        return self._pub.construction_order
```

Reply:
I corresponded.